examples/train_mnist_conv.py

```diff
@@ -35,54 +31,59 @@
     '9': 'Ankle boot'
 }
 
-display_tsne(train_images[:1000], train_labels[:1000], label_map)
 
 train_labels = get_one_hot(train_labels)
 test_labels = get_one_hot(test_labels)
 
-train_images = np.reshape(train_images, (train_images.shape[0], 28, 28))
-test_images = np.reshape(test_images, (test_images.shape[0], 28, 28))
-
-input_var = T.tensor4('input')
-y = T.fmatrix('truth')
-
-network_conv_config = {
-    'mode': 'conv',
-    'filters': [16, 32],
-    'filter_size': [[5, 5], [5, 5]],
-    'stride': [[1, 1], [1, 1]],
-    'pool': {
-        'mode': 'average_exc_pad',
-        'stride': [[2, 2], [2, 2]]
+train_images = tf.reshape(train_images, (train_images.shape[0], 28, 28))
+test_images = tf.reshape(test_images, (test_images.shape[0], 28, 28))
+
+
+with tf.Graph().as_default() as tf_graph:
```

Comment:
can all of these with statements be included in your net.py? it's easier than everyone having to remember them everytime/makes it cleaner.

Reply:
I think the only issue might be then we assume the reshape to be done on all incoming matrices which may not be the case. This would need to be generalized.

Reply:
Also, I realized that the datasets need to be fed as numpy rather TF tensors inorder to train the model, so I changed them back to np.reshape.